sys/include/net/gnrc/ipv6/nc.h

```diff
@@ -143,6 +146,15 @@ typedef struct {
      */
     vtimer_t nbr_adv_timer;
 
+#ifdef MODULE_GNRC_SIXLOWPAN_ND
+    vtimer_t rtr_sol_timer; /**< Retransmission timer for unicast router solicitations */
+#endif
+#ifdef MODULE_GNRC_SIXLOWPAN_ND_ROUTER
+    vtimer_t type_timeout;                  /**< Timer for type transissions */
+    eui64_t eui64;                          /**< the unique EUI-64 of the neighbor (might be
```

Comment:
Also, this is for router's only. I guess those are assumed to be a little more powerful than leaf-nodes (and they should, since they have to maintain a FIB too).